charts/rancher-monitoring/v0.0.8/values.yaml

```diff
@@ -376,6 +380,8 @@ prometheus:
     alertManagerNamespace: ""
   thanos:
     enabled: false
+  livenessProbe: {}
```

Comment:
Does the empty maps pass empty defaults to the corresponding charts?

Reply:
We are using `{{ merge .Values.livenessProbe .Values.defaultLivenessProbe | toYaml | indent 6 }}` in chart. We will merge `defaultLivenessProbe` and the empty (if users don't config it) `livenessProbe`

Reply:
I see, so the merge function just allows the user to override a specific field without having to pass the entire map?

Reply:
Yep